plugins/vpc/multipleSubnets.js

```diff
@@ -63,51 +70,42 @@ module.exports = {
 								]
 							}
 						]
-					}
+					};
 
-					if (!vpcId) {
-						results.push({
-							status: 3,
-							message: 'Unable to query for Subnets.',
-							region: region
-						});
-
-						return rcb();
-					}
-
-					ec2.describeSubnets(params, function(err, data) {
-						if (includeSource) source[region] = {error: err, data: data};
+					ec2.describeSubnets(params, function(subnetErr, subnetData) {
+						if (includeSource) source['describeSubnets'][region] = {error: subnetErr, data: subnetData};
 
-						if (err || !data || !data.Subnets) {
+						if (subnetErr || !subnetData || !subnetData.Subnets || !subnetData.Subnets.length) {
 							results.push({
 								status: 3,
-								message: 'Unable to query for Subnets.',
-								region: region
+								message: 'Unable to query for subnets in VPC.',
+								region: region,
+								resource: vpcId
 							});
 
 							return rcb();
 						}
 
-						if (data.Subnets.length > 2) {
+						if (subnetData.Subnets.length > 2) {
 							results.push({
 								status: 0,
-								message: 'There are ' + data.Subnets.length + ' different CIDR Blocks used in one VPC.',
+								message: 'There are ' + subnetData.Subnets.length + ' different subnets used in one VPC.',
 								region: region,
 								resource: vpcId
 							});
-						} else if (data.Subnets.length > 1) {
+						} else if (subnetData.Subnets.length > 1) {
 							results.push({
 								status: 1,
-								message: 'Using ' + data.Subnets.length + ' subnets may not be sufficient for a multi-layered architecture',
+								message: 'Using ' + subnetData.Subnets.length + ' subnets may not be sufficient for a multi-layered architecture',
 								region: region,
 								resource: vpcId
 							});
 						} else {
 							results.push({
 								status: 2,
-								message: 'Only one subnet, one VPC is used.',
+								message: 'Only one subnet (' + subnetData.Subnets[0].SubnetId + ') in one VPC is used.',
```

Comment:
Include the ID of the subnet in the message, but the VPC ID as the resource (allowing for easier suppressions)